src/lib/libembind.js

```diff
@@ -8,7 +8,7 @@
 /*global Module, asm*/
 /*global _malloc, _free, _memcpy*/
 /*global FUNCTION_TABLE, HEAP8, HEAPU8, HEAP16, HEAPU16, HEAP32, HEAPU32, HEAPF32, HEAPF64*/
-/*global readLatin1String*/
+/*global AsciiToString*/
```

Comment:
I don't actually know that these comments are event here for.. do you ?

Reply:
I think they were meant to be Eslint comments, but they are likely entirely useless in library code due to our custom syntax.

Reply:
We should be able to remove them in a separate PR, I don't think it's a blocker for this one.

Reply:
sgtm